curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657f0044be09db790b1eb1c5.md

```diff
@@ -7,15 +7,15 @@ dashedName: step-9
 
 # --description--
 
-Strings in pascal case start with a capital character. Since you've converted all such characters to lowercase and prepended an underscore to them, chances are, the converted snake case string has a dangling underscode at the start.
+In pascal case, strings begin with a capital letter. After converting all the characters to lowercase and adding an underscore to them, there's a chance of having an extra underscore at the start of your string.
 
-The easiest way to strip such unwanted character is by using the `.strip()` string method and passing an underscore to the method as argument.
+The easiest way to fix this is by using the `.strip()` string method and passing an underscore to the method as argument. This will remove any leading or trailing underscores from the string.
 
 ```py
 snake_cased_string.strip('_')
 ```
```

Comment:
Can we remove this and have or an example that can't be copied and pasted or a text explanation?